### ISSUE TITLE
Spec contains invalid syntax

### ISSUE THREAD

**Issue by mitar** [w3c/webappsec-suborigins#14]
Comments like:

```
<!----------------------------------------------------------------------------->
```

are not allowed. [See here](https://github.com/tabatkins/bikeshed/issues/574#issuecomment-174378760).

**Comment by devd** [comment]
hmm @metromoxie is the main user of those comments. I am happy to remove them if he is ok with that.

**Comment by metromoxie** [comment]
Huh, I had no idea! Happy to remove all of those. They're basically legacy from early specs I worked on.

**Comment by metromoxie** [comment]
Fixed by #18.
